src/main/java/net/dv8tion/jda/core/entities/MessageChannel.java

```diff
@@ -466,8 +473,15 @@ protected void handleResponse(Response response, Request request)
 
         if (message != null)
         {
-            body.field("content", message.getRawContent());
-            body.field("tts", message.isTTS());
+            if (message.getEmbeds().isEmpty())
+            {
+                body.field("content", message.getRawContent());
+                body.field("tts", message.isTTS());
+            }
+            else
+            {
+                body.field("payload_json", ((MessageImpl) message).toJSONObject().toString());
```

Comment:
Question: If `payload_json` exists, is there even a reason to use the above `body.field`s? Can we just send any form of message uploaded to Discord with a file in the `payload_json` or is it limited to only messages with embeds?